api/python/cellxgene_census/tests/experimental/util/test_csr_iter.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import pytest
+import scipy.sparse as sparse
+import tiledbsoma as soma
+
+import cellxgene_census
+from cellxgene_census.experimental.util import X_sparse_iter
+
+
+@pytest.fixture
+def small_mem_context() -> soma.SOMATileDBContext:
+    """used to keep memory usage smaller for GHA runners."""
+    cfg = {
+        "tiledb_config": {
+            "soma.init_buffer_bytes": 32 * 1024**2,
+            "vfs.s3.no_sign_request": True,
+        },
+    }
+    return soma.SOMATileDBContext().replace(**cfg)
+
+
+@pytest.mark.experimental
+@pytest.mark.live_corpus
+def test_X_sparse_iter(small_mem_context: soma.SOMATileDBContext) -> None:
+    with cellxgene_census.open_soma(census_version="stable", context=small_mem_context) as census:
```

Comment:
I recall we settled for running the tests on `latest`, since that could help catch issues with the builder. Is there any specific reason for running these on `stable`?

Reply:
just made a random choice.  Switch to latest as it does make sense!